src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Customizations/Skillsets/Models/NamedEntityCategory.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for NamedEntityCategory.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum NamedEntityCategory
+    {
+        [EnumMember(Value = "location")]
+        Location,
+        [EnumMember(Value = "organization")]
+        Organization,
+        [EnumMember(Value = "person")]
+        Person
+    }
+
+    [Obsolete]
```

Comment:
> @arv100kri Do you mean marking members with `[Obsolete]`? That isn't a breaking change in and of itself. And you're right; It doesn't matter anyway because we've bumped the major version number in anticipation of breaking changes for Complex Types.

Yup, and thanks for clarifying!

Reply:
Yup, I should've added the [Obsolete] to the enum and the extension... will do